protocol/persistence.go

```diff
@@ -856,7 +856,7 @@ func (db sqlitePersistence) ExpiredMessagesIDs(maxSendCount int) ([]string, erro
 			FROM
 				raw_messages
 			WHERE
-			message_type IN (?, ?) AND sent = ? AND send_count <= ?`,
+			(message_type IN (?, ?) OR resend_automatically) AND sent = ? AND send_count <= ?`,
```

Comment:
I think we want an index on `resend_automatically` if not there, also I think it's safer for release to do the fix as you did, but for develop we can stop checking the message type, and only trust `resend_automatically`, what do you think?

Reply:
Ok, i will add the index for release! As for dropping message_type checks in develop, would it work if i verify them and fix tests (if any broken with this PR) separately?

Reply:
Added index for `resend_automatically`.